package.json

```diff
@@ -30,11 +30,13 @@
   },
   "dependencies": {
     "cron-master": "~0.2.0",
+    "env-var": "~2.4.3",
     "later": "~1.2.0",
     "moment": "~2.17.1",
+    "rhmap-raygun-nodejs": "git+https://github.com/TkeITMobility/rhmap-raygun-nodejs.git",
```

Comment:
Make sure you reference a release tag in the URL. We don't want to always use the latest from master.